src/utils/convertMarkdown.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import ReactMarkdown from "react-markdown";
+import glm from "remark-gfm";
+import emoji from "remark-emoji";
+import sanitize from "rehype-sanitize";
+
+export const convertMarkdown = (text: string | undefined) => {
+  if (!text) return;
+  return text.split("\\n").map((word, index) => {
+    return (
+      <ReactMarkdown
+        linkTarget={"_blank"}
+        remarkPlugins={[[glm], [emoji]]}
+        rehypePlugins={[[sanitize]]}
+        key={index}
+      >
+        {word}
+      </ReactMarkdown>
+    );
+  });
+};
```

Comment:
What's the intent behind `text.split("\\n").map((word, index)` here?

---

This might make more sense as a custom component `RenderMarkdown` that wraps this up, rather than a 'util' function that returns JSX.